deploy.rb

```diff
@@ -8,6 +9,7 @@
 # ./deploy.rb stage
 
 require 'fileutils'
+require 'auditor'
```

Comment:
@jcoyne Should line 40 below also include some logic to set `SKIP_BUNDLE_AUDIT` in the IO env? That way we only run the bundle audit when we want to.

Reply:
done.